handlers/cni_network.go

```diff
@@ -109,11 +109,13 @@ func DeleteCNINetwork(ctx context.Context, cni gocni.CNI, client *containerd.Cli
 	container, containerErr := client.LoadContainer(ctx, name)
 	if containerErr == nil {
 		task, err := container.Task(ctx, nil)
-		log.Printf("[Delete] removing CNI network for %s\n", task.ID())
 		if err != nil {
-			return errors.Wrapf(err, "Unable to get task for container %s: %s", name, err)
+			log.Printf("[Delete] container %s does not have task\n", name)
```

Comment:
Should probably say `a task`, but how about: `Unable to find a task for container: %s\n`